quarkchain/cluster/cluster_config.py

```diff
@@ -167,6 +167,15 @@ class MonitoringConfig(BaseConfig):
     ERRORS = "error"
 
 
+class PrometheusConfig(BaseConfig):
+    TIMEOUT = 10
+    HOST = "http://localhost:38391"
+    GAP = 30
```

Comment:
What is GAP used for?

Reply:
It's the interval between two queries to avoid performance overload